tests/test_corpus.py

```
@@ -0,0 +1,99 @@
+    def test_an_empty_answer_is_still_recorded(self):
+        corpus = self.corpus()
+        path = corpus.record(jpeg(), payload={"results": []}, source="plate_recognizer")
+        sidecar = json.loads(path.with_suffix(".json").read_text())
+        self.assertEqual(sidecar["ocr"], {"results": []})
+
+    def test_oldest_pairs_are_pruned_once_the_size_bound_is_exceeded(self):
+        image = jpeg()
+        corpus = self.corpus(max_bytes=MIN_MAX_BYTES)
+        corpus._max_bytes = len(image) * 2 + 4096  # room for two pairs
+        paths = []
+        for index in range(4):
```

Ruff (0.16.3)

[warning] Loop control variable `index` not used within loop body

Rename unused `index` to `_index`

(B007)